cassiopeia/simulator/DataSimulator.py

```diff
@@ -0,0 +1,19 @@
+"""
+Abstract class DataSimulator, for overlaying data onto a CassiopeiaTree.
+
+All data simulators are derived classes of this abstract class, and at a minimum
+implement a method called `overlay_data`.
+"""
+import abc
+from cassiopeia.data import CassiopeiaTree
```

Comment:
nit: style -> add new line between `import abc` and `from cassiopeia.data import CassiopeiaTree`

Reply:
Thanks!